src/ctc/evm/erc20_utils/erc20_events.py

```diff
@@ -72,12 +76,11 @@ async def async_get_erc20_transfers(
     column = 'arg__amount'
     transfers = transfers.rename({old_column: column})
 
+    casted = np.array(transfers[column].to_list(), dtype=float)
```

Comment:
recasts the column from an "Object" type to a float type, which fixes downstream issues